source/plugins/go/src/oms.go

```diff
@@ -1281,42 +1284,77 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 			msgpBytes = msgp.AppendMapStrStr(msgpBytes, fluentForward.Entries[entry].Record)
 		}
 
-		if MdsdMsgpUnixSocketClient == nil {
-			Log("Error::mdsd::mdsd connection does not exist. re-connecting ...")
-			CreateMDSDClient(ContainerLogV2, ContainerType)
-			if MdsdMsgpUnixSocketClient == nil {
-				Log("Error::mdsd::Unable to create mdsd client. Please check error log.")
-
+		if IsWindows == true {
```

Comment:
@jatakiajanvi12 , we need to remove ingestionAuthToken code as this was added for MSI auth without AMA. lets discuss on this to figure out what code we need to remove.